packages/http-server/src/server.ts

```diff
@@ -80,23 +71,42 @@ export const createServer = (operations: IHttpOperation[], opts: IPrismHttpServe
 
       const { output } = response;
 
-      reply.code(output.statusCode);
-
-      if (output.headers) {
-        reply.headers(output.headers);
+      const inputOutputValidationErrors = response.validations.output
```

Comment:
is it `input` and `output` or only `output`?

Reply:
It's both. You can see they're concatenated.